sql/delete.go

```diff
@@ -39,7 +39,7 @@ func (p *planner) Delete(n *parser.Delete) (planNode, error) {
 	// deleting.
 	node, err := p.Select(&parser.Select{
 		Exprs: parser.SelectExprs{
-			&parser.StarExpr{TableName: parser.QualifiedName{tableDesc.Name}},
+			&parser.StarExpr{TableName: &parser.QualifiedName{Base: parser.Name(tableDesc.Name)}},
```

Comment:
`StarExpr` is still the only way to get a plain `*`. I'm looking into whether this can be further simplified.